app/src/main/java/org/keepgoeat/data/service/NaverAuthService.kt

```diff
@@ -25,7 +30,24 @@ class NaverAuthService @Inject constructor(
     fun loginNaver(loginListener: ((Boolean, Boolean) -> Unit)) {
         val oauthLoginCallback = object : OAuthLoginCallback {
             override fun onSuccess() {
-                // TODO: 네이버 로그인 성공 시 로직 처리
+                CoroutineScope(Dispatchers.Main).launch {
+                    val result = withContext(Dispatchers.IO) {
+                        NaverIdLoginSDK.getAccessToken()?.let { accessToken ->
+                            RequestAuth(
+                                accessToken, PLATFORM_NAVER
+                            )
+                        }?.let {
+                            authRepository.login(
+                                it
+                            )
+                        }
+                    }
+                    Timber.d(NaverIdLoginSDK.getAccessToken())
+                    loginListener(
+                        result?.type == SIGN_UP,
+                        localStorage.isClickedOnboardingButton
+                    )
+                }
```

Comment:
```suggestion
CoroutineScope(Dispatchers.Main).launch {
                    val result = withContext(Dispatchers.IO) {
                        val accessToken = requireNotNull(NaverIdLoginSDK.getAccessToken())
                        Timber.d(accessToken)
                        authRepository.login(RequestAuth(accessToken, PLATFORM_NAVER))
                    }

                    result?.let {
                        loginListener(it.type == SIGN_UP, localStorage.isClickedOnboardingButton)
                    }
                }
```

이렇게 바꾸면 좋을 것 같습니다~
result가 signup or signin이 아닌 null이 왔을 때도 아래 구문에서 false가 되어 홈 화면으로 이동할 것입니다..! null인 경우에는 화면 이동이 없도록 위와 같이 수정 부탁드려요~ 카카오도 마찬가지입니다! result 널체크 필수 !
```result?.type == SIGN_UP```